libs/filamat/src/eiff/BlobDictionary.h

```diff
@@ -35,12 +41,12 @@ class BlobDictionary {
     size_t getBlobCount() const;
 
     // Returns the total storage size, assuming that each blob is prefixed with a 64-bit size.
-    constexpr size_t getSize() const noexcept {
+	CONSTEXPR_IF_POSSIBLE size_t getSize() const noexcept {
```

Comment:
UTILS_CONSTEXPR would be better

Reply:
Maybe, but its not quite reflecting it's behavior.